ci/stf_tests.yml

```
@@ -0,0 +1,16 @@
+---
+- name: "Run e2e tests"
+  hosts: localhost
+  ignore_errors: true
+  environment:
+    PATH: "~/.crc/bin:~/.crc/bin/oc:~/bin:{{ ansible_env.PATH }}"
+    cifmw_rhol_crc_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
```

This most likely should be

```suggestion
    KUBECONFIG: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
```